examples/experimental/dagster-airlift/dagster_airlift/utils.py

```diff
@@ -3,8 +3,31 @@
 from typing import Optional
 
 DAGSTER_AIRLIFT_MIGRATION_STATE_DIR_ENV_VAR = "DAGSTER_AIRLIFT_MIGRATION_STATE_DIR"
+from typing import Generic, List, Mapping, TypeVar
+
+from dagster._record import record
+from dagster._serdes import whitelist_for_serdes
 
 
 def get_local_migration_state_dir() -> Optional[Path]:
     migration_dir = os.getenv(DAGSTER_AIRLIFT_MIGRATION_STATE_DIR_ENV_VAR)
     return Path(migration_dir) if migration_dir else None
+
+
+TKey = TypeVar("TKey")
+TValue = TypeVar("TValue")
+
+
+@whitelist_for_serdes
+@record
+class DictItem(Generic[TKey, TValue]):
```

Comment:
`@record` doesn't work with `Generic` on `3.8` , not clear if its solvable 

either way not much value over NamedTuple since we cant generate type check code for the resolved generics types (at least not yet)

Reply:
it was solvable https://github.com/dagster-io/dagster/pull/24777